core/services/keystore/chaintype/chaintype.go

```diff
@@ -17,10 +17,25 @@ const (
 	Solana ChainType = "solana"
 	// Terra for the Terra chain
 	Terra ChainType = "terra"
+	// StarkNet for the StarkNet chain
+	StarkNet ChainType = "starknet"
 )
 
+type ChainTypes []ChainType
+
+func (c ChainTypes) String() (out string) {
+	for i, chain := range c {
+		prefix := ", "
+		if i == 0 {
+			prefix = ""
+		}
+		out += prefix + string(chain)
+	}
+	return
```

Comment:
I wouldn't suggest `strings.Builder` just for the _efficiency_ in this case, but it also reads a little simpler too. WDYT?
```suggestion
	var sb strings.Builder
	for i, chain := range c {
		if i != 0 {
			sb.WriteString(", ")
		}
		sb.WriteString(string(chain))
	}
	return sb.String()
```

Too bad `strings.Join` doesn't 'just work' - maybe a future generic version could handle any underlying string type (`~string`).